public/app/features/dashboard/components/ShareModal/SharePublicDashboard/ConfigPublicDashboard/ConfigPublicDashboard.tsx

```diff
@@ -151,10 +151,9 @@ export function ConfigPublicDashboardBase({
             {...register('isPaused')}
             disabled={disableInputs}
             onChange={(e) => {
-              trackDashboardSharingActionPerType(
-                e.currentTarget.checked ? 'disable_sharing' : 'enable_sharing',
-                shareDashboardType.publicDashboard
-              );
+              reportInteraction('dashboards_sharing_public_pause_clicked', {
+                paused: e.currentTarget.checked,
+              });
```

Comment:
Is it necessary to track this? Wouldn't it be more useful to get the number of paused public dashboards from the DB?

Reply:
It is necessary if we want to know how the users interact with this (how many times they click this toggle, DB will only show the current value) but I didn't really think much about it, it's just a refactor.